vimdoc/output.py

```diff
@@ -25,7 +25,7 @@ def Filename(self):
 
   def Write(self):
     filename = os.path.join(self.docdir, self.Filename())
-    with io.open(filename, 'w', encoding='utf-8') as self.file:
+    with io.open(filename, 'w', encoding='utf-8', newline='\n') as self.file:
```

Comment:
because people may use vimdoc in both window and linux os for same project, if the generation txt file are different, it will cause diff patch when using git.

Reply:
Done. Can you check my wording is clear?

Reply:
Sure, lgtm. (Probably anything that calls out that we're doing this deliberately is fine.)